plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSplitManager.java

```diff
@@ -172,12 +197,23 @@ private AddFileEntry addFileEntryOfSize(long fileSize)
         return new AddFileEntry(FILE_PATH, ImmutableMap.of(), fileSize, 0, false, Optional.empty(), Optional.empty(), ImmutableMap.of());
     }
 
+    private AddFileEntry addFileEntryOfSizeForAbsolutePath(long fileSize)
+    {
+        return new AddFileEntry(ABSOLUTE_FILE_PATH, ImmutableMap.of(), fileSize, 0, false, Optional.empty(), Optional.empty(), ImmutableMap.of());
+    }
+
     private DeltaLakeSplit makeSplit(long start, long splitSize, long fileSize, double minimumAssignedSplitWeight)
     {
         SplitWeight splitWeight = SplitWeight.fromProportion(Math.min(Math.max((double) fileSize / splitSize, minimumAssignedSplitWeight), 1.0));
         return new DeltaLakeSplit(FULL_PATH, start, splitSize, fileSize, Optional.empty(), 0, ImmutableList.of(), splitWeight, TupleDomain.all(), ImmutableMap.of());
     }
 
+    private DeltaLakeSplit makeSplitForAbsolutePath(long start, long splitSize, long fileSize, double minimumAssignedSplitWeight)
```

Comment:
Can we de-duplicate some of the code between here and the `makeSplit` method? Same for `addFileEntryOfSizeForAbsolutePath` and `addFileEntryOfSize`

Reply:
this is done